.github/workflows/ios_libs.yml

```diff
@@ -0,0 +1,216 @@
+              target="ios-sim-cross-x86_64"
+          fi
+          ./build-libssl.sh --targets="$target" --version="$OPENSSL_VERSION" --deprecated
+
+          # Cleanup
+          cd ..
+          mv src/lib src/include .
+          rm -rf src
+
+      - name: Build Zlib
+        run: |
+          # Download Zlib
+          wget "https://zlib.net/zlib-$ZLIB_VERSION.tar.gz" -O /tmp/zlib.tar.gz
+
+          # Setup build directory
+          mkdir -p "/tmp/build/zlib/$ARCH/src"
+          tar xvf /tmp/zlib.tar.gz -C "/tmp/build/zlib/$ARCH/src" --strip-components=1
+          cd "/tmp/build/zlib/$ARCH/src"
+
+          # Build Zlib
+          CFLAGS='-O3' \
+          CPPFLAGS='-O3' \
+          ${{ github.workspace }}/build/scripts/configure-ios.sh \
+            "$ARCH" \
+            --static \
+            --prefix="/tmp/build/zlib/$ARCH"
+          make -j$(sysctl hw.ncpu | awk '{print $2}') install
+
+          # Cleanup
+          cd ..
+          rm -rf src share lib/pkgconfig
+
+      - name: Build Libevent
+        run: |
+          # Download Libevent
+          wget "https://github.com/libevent/libevent/releases/download/release-$LIBEVENT_VERSION/libevent-$LIBEVENT_VERSION.tar.gz" -O /tmp/libevent.tar.gz
+
+          # Setup build directory
+          mkdir -p "/tmp/build/libevent/$ARCH/src"
+          tar xvf /tmp/libevent.tar.gz -C "/tmp/build/libevent/$ARCH/src" --strip-components=1
+          cd "/tmp/build/libevent/$ARCH/src"
+
+          # Build Libevent
+          if [[ "$ARCH" == 'arm64' || "$ARCH" == 'arm64e' ]]; then
+              host='aarch64-apple-darwin*'
+          elif [[ "$ARCH" == 'x86_64' ]]; then
+              host='x86_64-apple-darwin*'
+          fi
+          CFLAGS="-O3 -I/tmp/build/openssl/$ARCH/include" \
+          CPPFLAGS="-O3  -I/tmp/build/openssl/$ARCH/include" \
+          LDFLAGS="-L/tmp/build/openssl/$ARCH/lib" \
+          ${{ github.workspace }}/build/scripts/configure-ios.sh \
+            "$ARCH" \
+            --disable-shared \
+            --disable-samples \
+            --enable-function-sections \
+            --host="$host" \
+            --prefix="/tmp/build/libevent/$ARCH"
+          make -j$(sysctl hw.ncpu | awk '{print $2}') install
+
+          # Cleanup
+          cd ..
+          rm -rf src bin $(ls -1ad lib/* | grep -v 'libevent.a')
+
+      - name: Compress build directory
+        if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') }}
+        run: tar czvf "libs-$ARCH.tar.gz" -C "/tmp" build
+
+      - name: Upload build archive
+        if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: libs-${{ github.run_id }}-ios
+          path: libs-${{ matrix.archs }}.tar.gz
+
+  release:
+    name: Create release and bundle assets
+    needs: build
+    if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') }}
+    runs-on: macos-latest
```

Comment:
I use `lipo` in this job, I don't know if there is a difference between the Linux and macOS versions, especially when it comes to merge libs intended for iOS.

The amount of runners is sufficient since we don't have a lot of pull request nor merge on master (and this job is not triggered by pull request anyway).

Reply:
:ok_hand: